src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public static String getRandomColor() {
```

Comment:
```suggestion
    private static final Random random = new Random()

    public static String getRandomColor() {
```'

this is how you create global variables